sopel/bot.py

```diff
@@ -659,38 +663,73 @@ def dispatch(self, pretrigger):
         for priority in ('high', 'medium', 'low'):
             items = self.get_triggered_callables(priority, pretrigger, blocked)
             for func, trigger, is_blocked in items:
+                function_name = "%s.%s" % (func.__module__, func.__name__)
                 # skip running blocked functions, but track them for logging
                 if is_blocked:
-                    function_name = "%s.%s" % (
-                        func.__module__, func.__name__
-                    )
                     list_of_blocked_functions.append(function_name)
                     continue
 
                 # call triggered function
                 wrapper = SopelWrapper(
                     self, trigger, output_prefix=func.output_prefix)
                 if func.thread:
+                    # run in a separate thread
                     targs = (func, wrapper, trigger)
                     t = threading.Thread(target=self.call, args=targs)
+                    t.name = '%s-%s' % (t.name, function_name)
                     t.start()
+                    running_triggers.append(t)
                 else:
+                    # direct call
                     self.call(func, wrapper, trigger)
 
+        # log blocked functions
         if list_of_blocked_functions:
             if nick_blocked and host_blocked:
                 block_type = 'both'
             elif nick_blocked:
                 block_type = 'nick'
             else:
                 block_type = 'host'
-            LOGGER.info(
-                "[%s]%s prevented from using %s.",
+            LOGGER.debug(
+                "[%s] %s prevented from using %s.",
```

Comment:
This honestly confused the #*!$ out of me when I was testing something else earlier, so a PR where the line is already touched is a perfect place to suggest a different way of putting it.

```suggestion
                "%s prevented from using %s by %s.",
```

Unfortunately this isn't useful as a suggestion because it'll require changes to several surrounding lines:

* The if/elif/else of block types needs to change the strings:
  * `block_type = 'both blocklists'`
  * `block_type = 'nick blocklist'`
  * `block_type = 'host blocklist'`
* The order of parameters passed to the function needs to be updated below

I don't think this will make the eventual ignore system improvements (#1355) any harder to implement.

Reply:
That's all fair to me, will do. :)